client/deposits/instant-deposits/test/index.tsx

```diff
@@ -92,7 +92,9 @@ describe( 'Instant deposit button and modal', () => {
 			screen.queryByRole( 'dialog', { name: /instant deposit/i } )
 		).not.toBeInTheDocument();
 		fireEvent.click(
-			screen.getByRole( 'button', { name: /Deposit available funds/i } )
+			screen.getByRole( 'button', {
+				name: /Instantly deposit \$123\.45/i,
+			} )
```

Comment:
> Do we need to remove [this test](https://github.com/Automattic/woocommerce-payments/blob/86b2095b0a7fdf753af0edcd742f17fca0c46306/client/deposits/instant-deposits/test/index.tsx#L73-L74) that checks how the instant deposit button looks when amount is 0? As [per logic ](https://github.com/Automattic/woocommerce-payments/blob/86b2095b0a7fdf753af0edcd742f17fca0c46306/client/components/account-balances/index.tsx#L131-L132)of the parent component, the button will be only displayed if the amount is greater than 0.

> I personally think the test is redundant, but wanted a confirmation from another eye since this is about removing a test.


I agree, I don't think keeping that test is a priority. 

Tip: it's easier to review stuff like this with a review comment thread. Some options:

- Add a self-review comment on the relevant file so the reviewer sees it roughly in context.
- Add a comment to the relevant code, then comment on that in a self-review.
- Best: be bold, delete the test, and make your pitch in your PR! You can always revert.

Reply:
Awesome tips! Rua! Thank you!

Reply:
Test removed - [7034e22](https://github.com/Automattic/woocommerce-payments/pull/8970/commits/7034e224a702485e434b4a25b03dccba0d4c62a2) . Thank you!